docs/docs/concepts/components/overview.mdx

```diff
@@ -1,62 +1,102 @@
+The `assistant_id` key should be a unique value and allows you to distinguish multiple
+deployed assistants.
+This id is added to each event's metadata, together with the model id.
+See [event brokers](../production/event-brokers.mdx) for more information.
+Note that if the config file does not include this required key or the placeholder default value 
+is not replaced, a random assistant name will be generated and added to the configuration 
+every time you run `rasa train`.
 
-TODO: update
 
-You can leave the pipeline and/or policies key out of your configuration file.
-When you run `rasa train`, the Suggested Config feature will select a default configuration
-for the missing key(s) to train the model.
 
-Make sure to specify the language key in your `config.yml` file with the
-2-letter ISO language code.
+## Pipeline 
 
-Example `config.yml` file:
+The `pipeline` key lists the components which will be used to process and understand the messages
+that end users send to your assistant. 
+In a CALM assistant, the output of your pipeline components is a list of [commands](../dialogue-understanding.mdx).
 
-```yaml-rasa (docs/sources/data/configs_for_docs/example_for_suggested_config.yml)
+The main component in your pipeline is the `LLMCommandGenerator`.
+Here is what an example configuration looks like:
+
+```yaml-rasa title="config.yml"
+pipeline:
+  - name: LLMCommandGenerator
+    llm:
+      model_name: "gpt-4"
+      request_timeout: 7
+      temperature: 0.0
 ```
 
-The selected configuration will also be written as comments into the `config.yml` file,
-so you can see which configuration was used. For the example above, the resulting file
-might look e.g. like this:
+The full set of configurable parameters is listed [here](../dialogue-understanding.mdx).
+
+All components which make use of LLMs have common configuration parameters which are listed [here](../llm-configuration.mdx)
 
-```yaml-rasa (docs/sources/data/configs_for_docs/example_for_suggested_config_after_train.yml)
+
+### Combining CALM and NLU-based components
+
+<RasaProLabel />
+
+Rasa Pro allows you to combine both NLU-based and CALM components in your pipeline. 
+See a full list of NLU-based components [here](../nlu-based-assistants/components.mdx).
+
+## Policies
+
+The `policies` key lists the [dialogue policies](../policies.mdx) your assistant will use 
+to progress the conversation. 
+
+```yaml-rasa title="config.yml"
+policies:
+  - name: rasa.core.policies.flow_policy.FlowPolicy
 ```
 
-If you like, you can then un-comment the suggested configuration for one or both of the
-keys and make modifications. Note that this will disable automatic suggestions for this
-key when training again.
-As long as you leave the configuration commented out and don't specify any configuration
-for a key yourself, a default configuration will be suggested whenever you train a new
-model.
+The `FlowPolicy` currently doesn't have an additional configuration parameters.
```

Comment:
do we have a doc of the `FlowPolicy` in `policies.mdx`? I would add a link there